snmp/datadog_checks/snmp/data/profiles/_base_dell.yaml

```diff
@@ -0,0 +1,264 @@
+# Base profile for DELL devices
```

Comment:
It might be better to treat this file as a "profile mixin" that functions as "Provide metrics that can be found in Dell devices that are or contain DRAC components".

i.e.

```yaml
# _dell-rac.yaml

metrics:
  # ...
```

And then update the "concrete" profiles so that _they_ extend from `_base.yaml` and include this mixin:

```yaml
# dell-poweredge.yaml

sysobjectid: ...

extends:
  - _base.yaml
  - _dell-rac.yaml  # These metrics are known to be available in Poweredge r740 devices.

metrics:
  # ...
```

```yaml
# idrac.yaml

sysobjectid: ...

extends:
  - _base.yaml
  - _dell-rac.yaml
```

This is the pattern we have for eg `_generic-if.yaml`, which doesn't extend from anything and can be included in profiles whenever we know the device exposes IF-MIB OIDs.